plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/model/MavenCoordinates.java

```diff
@@ -28,6 +28,16 @@
   public static final String JAR_TYPE = "jar";
   public static final String MAVEN_CENTRAL_REPO = "central";
 
+  public static MavenCoordinates coordinates(String groupId, String artifactId) {
```

Comment:
These methods may be a little too error prone to include in the main, non-test API. In particular, nothing breaks if you swap the groupID and artifactId arguments.

Reply:
ok